src/app/nav-bar/nav-bar.component.html

```diff
@@ -11,4 +11,7 @@
       <span class="glyphicon glyphicon-glass"></span>
       <p class="menu-item-description">Make Cocktail</p>
     </li>
+    <li class="menu-item" (click)="logout()">
+      <p class="menu-item-description">Logout</p>
```

Comment:
we need some icon for this because this item won't be seen on small screens

Reply:
for example `<span class="glyphicon glyphicon-log-out"></span>`

Reply:
I will use ```<span class="glyphicon glyphicon-log-out"></span>```